recipes/sos-papermill/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sos-papermill" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ccb24877a5773a7d1d03361b77f23904e3dc266b459eb1439505ba539aa592a
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sos-notebook
+    - python >=3.6
+    - papermill
+
+test:
+  requires:
+    - nose
+    - pytest
+    - nbformat
+    - mock
+  imports:
+    - sos_papermill
+
+about:
+  home: https://github.com/vatlab/sos-papermill
+  license: BSD
```

Comment:
This should be something like "BSD-3-Clause"

Reply:
fixed